components/alert/alert.scss

```diff
@@ -86,6 +99,18 @@
     }
   }
 
+  &__alert-tag {
```

Comment:
So we are starting to use these badges/tags more often, should we create a component that we reuse in the solution card and alerts?

One more thing, I think we should name the badges and not tags so "__alert-badge" as this is what they are called in the MP solution and we should probably align

Reply:
Updated the tag to badge